job/common/src/main/java/alluxio/job/wire/PlanInfo.java

```diff
@@ -225,13 +226,14 @@ public boolean equals(Object o) {
         && Objects.equal(mResult, that.mResult)
         && Objects.equal(mLastUpdated, that.mLastUpdated)
         && Objects.equal(mName, that.mName)
-        && Objects.equal(mDescription, that.mDescription);
+        && Objects.equal(mDescription, that.mDescription)
+        && Objects.equal(mAffectedPaths, that.mAffectedPaths);
```

Comment:
Is this coming from `MoveConfig`? Will it be sorted always?

Reply:
not necessarily, but we insert them into the list in certain order, so they should stay in that order and this comparison would work. 
In most cases, this list has one element only.